dev/test/bulk-writer.ts

```diff
@@ -372,12 +376,35 @@ describe('BulkWriter', () => {
     const doc = firestore.doc('collectionId/doc');
     bulkWriter.set(doc, {foo: 'bar'}).catch(err => {
       incrementOpCount();
+      expect(err instanceof BulkWriterError).to.be.true;
       expect(err.code).to.equal(Status.DEADLINE_EXCEEDED);
     });
 
     return bulkWriter.close().then(async () => verifyOpCount(1));
   });
 
+  it('throws UnhandledPromiseRejection if the error is not caught', async () => {
+    let errorThrown = false;
+    const unhandledDeferred = new Deferred<void>();
+    process.on('unhandledRejection', () => {
+      errorThrown = true;
+      unhandledDeferred.resolve();
+    });
```

Comment:
Let's confirm with Yoshi team whether it would be ok to swallow these.

Reply:
SG, started a conversation with them in the node room.